src/main/scala/util/package.scala

```diff
@@ -234,23 +234,55 @@ package object util {
     map.view.map({ case (k, vs) => k -> vs.toList }).toList
   }
 
+/** provides operators useful for working with bidirectional [[Bundle]]s
+  * 
+  * In terms of [[Flipped]] with a producer 'p' and 'consumer' c:
+  * c :<= p // means drive all unflipped fields of 'c' from 'p' (e.g.: valid/bits)  
+  * c :=> p // means drive all flipped fields of 'c' from 'p' (e.g.: ready)  
+  * p :<> c // do both of the above
+  * c :<> p // do both of the above, but you'll probably get a Gender error later.
```

Comment:
Pretty sure you meant:

  * c :<> p // do both of the above
  * p :<> c // do both of the above, but you'll probably get a Gender error later.
Write